cmd/manager/main.go

```diff
@@ -127,8 +127,14 @@ func main() {
 
 	log.Info("Starting the Cmd.")
 
+	stopCH, err := k8s.StartStopSignalHandler(mgr.GetClient())
```

Comment:
can we pass correct namespace here?
```suggestion
	stopCH, err := k8s.StartStopSignalHandler(mgr.GetClient(), namespace)
```
there are a few cases that should be tested:
1. `WATCH_NAMESPACE` can be equal to an empty string - just pass it as it is to `client.Get`, which should return objects in all namespaces
2. `WATCH_NAMESPACE` can be a comma-separated list `strings.Split(namespace, ",")` - it is better to iterate over it
3. `WATCH_NAMESPACE` can have one namespace name - just pass it as it is to `client.Get`, which should return objects in all namespaces